pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/JuniperShipKitRoute.java

```diff
@@ -0,0 +1,320 @@
+package org.broadinstitute.dsm.route;
+
+import java.sql.Connection;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicReference;
+
+import com.easypost.exception.EasyPostException;
+import com.easypost.model.Address;
+import lombok.NonNull;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.dsm.DSMServer;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.InstanceSettings;
+import org.broadinstitute.dsm.db.KitRequestShipping;
+import org.broadinstitute.dsm.db.dao.ddp.instance.DDPInstanceDao;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.settings.InstanceSettingsDto;
+import org.broadinstitute.dsm.model.KitRequest;
+import org.broadinstitute.dsm.model.KitRequestSettings;
+import org.broadinstitute.dsm.model.KitType;
+import org.broadinstitute.dsm.model.KitUploadObject;
+import org.broadinstitute.dsm.model.KitUploadResponse;
+import org.broadinstitute.dsm.model.Value;
+import org.broadinstitute.dsm.security.RequestHandler;
+import org.broadinstitute.dsm.util.DDPKitRequest;
+import org.broadinstitute.dsm.util.EasyPostUtil;
+import org.broadinstitute.dsm.util.KitUtil;
+import org.broadinstitute.dsm.util.externalshipper.ExternalShipper;
+import org.broadinstitute.lddp.handlers.util.Result;
+import org.broadinstitute.lddp.util.DeliveryAddress;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+
+public class JuniperShipKitRoute extends RequestHandler {
+
+    private static final Logger logger = LoggerFactory.getLogger(JuniperShipKitRoute.class);
+
+    @Override
+    protected Object processRequest(Request request, Response response, String userId) throws Exception {
+        return null;
+    }
+
+    public Object createAJuniperKit(List<KitRequest> kitUploadObjects, String studyGuid, String kitTypeName,
```

Comment:
To echo Zim's point elsewhere, would createKit() work here instead?
Improves flexibility in case you want to use this more broadly, and I assume there is nothing particular to Juniper in this method?

Reply:
Perhaps a nit, but articles (the, a) are pretty much never used when naming things (at least in the software I've seen). So perhaps `createNonPepperKit`?